src/modules/legacyReferrals/components/ProjectLegacyReferrals.tsx

```diff
@@ -0,0 +1,105 @@
+import AddIcon from '@mui/icons-material/Add';
+import { Stack } from '@mui/material';
+
+import ProjectOutgoingReferralPostingsTable from './ProjectOutgoingReferralPostingsTable';
+import ProjectReferralPostingsTable from './ProjectReferralPostingsTable';
+import ProjectReferralRequestsTable from './ProjectReferralRequestsTable';
+
+import ButtonLink from '@/components/elements/ButtonLink';
+import TitleCard from '@/components/elements/TitleCard';
+import { useGlobalFeatureFlags } from '@/hooks/useGlobalFeatureFlags';
+import { useProjectDashboardContext } from '@/modules/projects/components/ProjectDashboard';
+import { ProjectDashboardRoutes } from '@/routes/routes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+/**
+ * This component is used to show the legacy referrals page.
+ *
+ * If the project has CE referrals enabled, it disables the ability to create
+ * new legacy referrals and referral requests.
+ * TODO(#8142) fully sunset legacy referrals, remove this tab
+ */
+const ProjectLegacyReferrals = () => {
```

Comment:
UI unchanged aside from disabling action buttons in "legacy mode". Copied from the deleted `ProjectReferralsPage`.